_overviews/scala3-book/first-look-at-types.md

```diff
@@ -239,7 +239,7 @@ For example:
 
 ```scala
 val x: Long = 987654321
-val y: Float = x  // 9.8765434E8 (note that some precision is lost in this case)
+val y: Float = x.toFloat  // 9.8765434E8 (note implicit conversion is depreciated since 2.13.1 when it would result in percision loss)
```

Comment:
Thank you for the fix. There is typo in “deprec*i*ated” :)

However, I think this example should be moved below. It is better to first show the cast from `Char` to `Int`, which corresponds to a simple case of type widening.

We could maybe add another example (e.g. `Float` to `Double`?).

Then, there is this section below (from line 248 to 255) where we explain that casting to a narrower type does not work. I think we should move the `Long` to `Float` example here, and we should mention that it is also possible to cast from `Float` to `Long` by explicitly calling `toLong`.

Reply:
Great suggestions and feedback, thank you so much, and great catch on the typo.

I've updated, taking your feedback into consideration. I opted for a Byte -> Int example vs the suggested Float -> Double to avoid any confusion the increased precision might cause.
```scala
val f: Float = 3.14
val d: Double = f // 3.140000104904175
```